### ISSUE TITLE
Angular Error: Type 'string' is not assignable to type 'HTMLElement'

### ISSUE THREAD

**Issue by ericrovtar** [farengeyt451/ngx-tippy-wrapper#7]
Hi!

First, thanks for an awesome Angular wrapper to Tippy.js.

I'm testing it out in our application, and I'm getting an Angular compile error when I use it: `Type 'string' is not assignable to type 'HTMLElement'.`

I've tried using Tippy on a `span` and a `button` but I keep getting this error.

Our application is running Angular `9.1.3`, so I'm not sure if that's the issue. If so, can you recommend which version of your wrapper I should use?

Thanks!

-Eric

**Comment by farengeyt451** [comment]
Hi @ericrovtar! Can u please attach your code example or recreate issue in **codesandbox** or **stackblitz**. I dont think that your problem in Angular versinon. This demo app https://s8q4n.csb.app/ running on Angular `9.1.12`, with no errors.

**Comment by ericrovtar** [comment]
@farengeyt451 I did try recreating it in your codesandbox, but I didn't see the same compile errors. 

My usage seems pretty basic. It's four examples like this with different icons/text:

```
<span class="[ fa fa-fw fa-people-carry ] [ text--icon ] [ localizejs ]"
  ngxTippy data-tippy-content="This Waste Stream has a Service Provider."
  [tippyProps]="{ placement: 'left' }"></span>```

The full error is:

```
src/app/waste-stream/list/list-waste-stream.component.html:116:25 - error TS2322: Type 'string' is not assignable to type 'HTMLElement'.

116                         ngxTippy data-tippy-content="This Waste Stream has a Service Provider."
                            ~~~~~~~~

  src/app/waste-stream/list/list-waste-stream.component.ts:25:18
    25     templateUrl: './list-waste-stream.component.html'
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component ListWasteStreamComponent.```

The application still works and runs, but I know it won't finish compiling for deployment.

I also tested something like this and still get the same error:

```
<div ngxTippy data-tippy-content="Test">This is just a test</div>```

I should note that it's always underlining the `ngxTippy` directive.

If I can test something else, please let me know. I'd love to get this working for our application as we look to replace another library.

Thanks!

**Comment by farengeyt451** [comment]
@ericrovtar One more thing, which version of **typescript** do you use in your project?

**Comment by ericrovtar** [comment]
@farengeyt451 We're on 3.8.3.

**Comment by farengeyt451** [comment]
I recreated project on angular `9.1.3` and typescript `3.8.3` and did not see any errors. 
Maybe you have some specific tempalte parse settigs? Try to pass content for tooltip through service or through [tippyProps] binding. I have not any other suggestions, why this error occurs.

**Comment by ericrovtar** [comment]
@farengeyt451 Thanks. I'll keep testing and see if I can figure it out. Thanks for helping me know it's not an Angular or TypeScript issue.

**Comment by farengeyt451** [comment]
Hi @ericrovtar. I found the problem, in your project probably enabled `strictTemplates` flag. This causes @Input strict check and as a result error in library's typings. I will fix this in near future and rollup new package.
As temporary design u can use empty directive binding:

```
<button [ngxTippy] data-tippy-content="Tooltip content">
  Element with tooltip
</button>
```

**Comment by farengeyt451** [comment]
Fixed on `v2.1.0`
